org.hl7.fhir.dstu3/src/main/java/org/hl7/fhir/dstu3/model/ActivityDefinition.java

```diff
@@ -4190,8 +4190,8 @@ public Timing getTimingTiming() throws FHIRException {
       return (Timing) this.timing;
     }
 
-    public boolean hasTimingTiming() { 
-      return this != null && this.timing instanceof Timing;
+    public boolean hasTimingTiming() {
+      return this.timing != null && this.timing instanceof Timing;
```

Comment:
Saying `this.timing != null` is redundant... An `instanceof` check will always return false if the left hand side is null.